modules/services/cloud-bench/versions.tf

```diff
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = ">= 4.21.0"
+      version = "< 4.65.0"
     }
     google-beta = {
```

Comment:
We also need the `google-beta` version to be pinned as well, otherwise that hits the same error too